modules/fscloud/variables.tf

```diff
@@ -19,6 +19,12 @@ variable "region" {
   description = "The IBM Cloud region where the cluster will be provisioned."
 }
 
+variable "use_private_endpoint" {
+  type        = bool
+  description = "Enable if the runtime has to have access to private endpoints."
```

Comment:
I'd specify exactly what is meant by "runtime".

Suggest to use "Enable if the Terraform execution environment requires access to private endpoints."